cmd/operator-sdk/generate/internal/genutil.go

```diff
@@ -55,18 +52,6 @@ func IsPipeReader() bool {
 	return info.Mode()&os.ModeNamedPipe != 0
 }
 
-// PluginKeyToOperatorType converts a plugin key string to an operator project
-// type.
-// TODO(estroz): this can probably be made more robust by checking known
-// plugin keys directly.
-func PluginKeyToOperatorType(pluginKey string) projutil.OperatorType {
-	switch {
-	case strings.HasPrefix(pluginKey, "go"):
-		return projutil.OperatorTypeGo
-	}
-	return ""
-}
-
```

Comment:
move the implementation to the project util since is required/useful to check the project type.